reference/perf-benchmarks.md

```diff
@@ -15,3 +15,7 @@ The following benchmarks give an overview of the performance that you can achiev
 |---------------|---------------------|----------------------|------------|---------------------|--------------|---------|
 | m6g.2xlarge   | 1.11.2 (appliance)  | software / software  | 720p       | Android 10 Launcher | 3            | 20      |
 | m5a.2xlarge   | 1.11.2 (appliance)  | software / software  | 720p       | Android 10 Launcher | 3            | 13      |
+| g5g.metal     | 1.12.1 (appliance)  | hardware / hardware  | 720p       | [BombSquad](https://www.froemling.net/apps/bombsquad) | 55 | 28 |
+| g5g.metal     | 1.12.1 (appliance)  | hardware / hardware  | 720p       | [BombSquad](https://www.froemling.net/apps/bombsquad) | 40 | 55 |
```

Comment:
What's the key factor which yields out different numbers of containers and avg FPS when rendering and encoding both are based on hardware and instance-type/appliance version are identical in the above two items.

Reply:
We have a different target FPS. In the first case the target is 30, in the second case it is 60. So the two lines show that you can run higher number of containers by accepting lower frame rates and running with higher frame rates when accepting a lower number of containers.

Reply:
Then we need a column like `Requested FPS` as another factor that people can differentiate those two items, otherwise it introduces confusion.

Reply:
So "Avg. FPS" is a value that you specify?
This is a bit misleading. The way the table is ordered, and the fact that it's an average, makes it seem like a result. So that with the settings from the first 5 columns, you achieve the results in the last two columns.

Reply:
The target FPS is more an implementation detail to stabilize the frame rate across a set of containers. The difference here is that the more clients per GPU you have the less FPS each of them can achieve. That's independent if we limit the FPS or not. So with 40 clients you can achieve higher frame rates than with 55. 

The "Avg FPS" column is the result, not an input. If we add another column specifying an input "Target FPS" I don't know if that actually helps to understand the results as in the end it's just an implementation detail to get more stablized FPS values across a set of clients an not one at 50 and one at 20.

Reply:
I think we need to add something - because right now, the only thing you can see from the table is that you can get different results with the same setup, but you don't know how.